app/App/Panel/Main/Account/Requests/TransactionRequest/TxFeeNew/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import Restore from 'react-restore'
+import BigNumber from 'bignumber.js'
+// import svg from '../../../../../../../../resources/svg'
+
+BigNumber.config({ ROUNDING_MODE: BigNumber.ROUND_DOWN })
```

Comment:
@floating does this change the config for every instance of BigNumber that we create?

Reply:
Hmm, not sure but it's not needed anyway so I will remove